lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -1414,7 +1418,7 @@ ThreadContext::SetForceOneIdleCollection()
 BOOLEAN
 ThreadContext::IsOnStack(void const *ptr)
 {
-
+#ifdef _WIN32
```

Comment:
Windows code either reading ```FS``` or calling ```GetCurrentThreadStackLimits```. Those are platform behavior, not VC support. On Linux we call ```GetCurrentThreadStackBounds``` which is currently declared in CommonPal.h under ```!_WIN32```. Looks _WIN32 is more appropriate here?